### ISSUE TITLE
Notification version feature use alpha/beta update

### ISSUE THREAD

**Issue by AlexandrePavy** [Sylius/Sylius#10919]
Hello all, 

Firstly this is nice project and he more on more awesome, i'm sorry if this not right place.

I'm using notification update on admin dashboard, but http://gus.sylius.com/version display alpha/beta version (eg : `{"version":"1.7.0-ALPHA.2"}`)

Maybe should be good to exclude alpha/beta version, or set as setting or when not in prod env ?

Best regards !

**Comment by Zales0123** [comment]
Agreed, IMO we should only notify about stable minor/major releases by default 👍

**Comment by gponty** [comment]
Agree with you
